dm_cli/utils.py

```diff
@@ -172,7 +172,7 @@ def replace_relative_references(
                     )
                 )
             return extends_list
-        if key == "attributeType" and value in PRIMITIVES:
+        if key == "attributeType" and (value in PRIMITIVES or value == BuiltinDataTypes.OBJECT.value):
```

Comment:
Since "BuiltinDataTypes" is a superset of PRIMITIVES, it should be enough to only check the latter